doc/CodeDocumentation.conf.in

```diff
@@ -179,6 +201,16 @@ SHORT_NAMES            = NO
 
 JAVADOC_AUTOBRIEF      = NO
```

Comment:
While we're thinking about Doxygen, do we want to enable [autobrief](https://www.doxygen.nl/manual/config.html#cfg_javadoc_autobrief)? This means we won't have to write `@brief` all the time (and has few downsides as far as I can tell).

Reply:
I like that suggestion 👍

Reply:
While I am not opposed to the idea I would like to leave this to a separate PR as this is not so trivial to change. Take the docs for `NodesUpdated` as an example, where "e.g." is used, which is interpreted as  the end of the brief part.

Reply:
OK, let's keep it as a separate PR.

I assumed that you could "override" the autobrief docs with an explicit `@brief`, but it appears that doesn't work.

Another (better?) option would be to enable `MULTILINE_CPP_IS_BRIEF`. This would make the first paragraph of a `///` comment block the brief comment, and subsequent paragraphs would be the detailed description.